src/app/components/dashboard/dashboard.component.ts

```diff
@@ -1,12 +1,22 @@
 import { Component} from '@angular/core';
-import { DataTableModule } from 'primeng/primeng';
+import { Observable } from 'rxjs';
+
+import { ReturnSetInfoDataService } from './dashboard.service';
 
 @Component({
   selector: 'dashboard',
   templateUrl: './dashboard.component.html',
-  styleUrls: ['./dashboard.component.scss']
+  styleUrls: ['./dashboard.component.scss'],
+  providers: [ ReturnSetInfoDataService ]
 })
-export class DashboardComponent {  
-  constructor() {}  
-  
+export class DashboardComponent {
+  data: Observable<Array<any>>;
+
+  constructor(private dataService: ReturnSetInfoDataService) {
+    dataService.getSetInfoData().subscribe(res => {
+      this.data = res;
+      debugger
```

Comment:
debugger!!!!!

Reply:
@nishintn 

Here also same. The debugger was removed and the code is not seen in my local.